plugins/Actions/VisitorDetails.php

```diff
@@ -160,6 +161,12 @@ public function extendActionDetails(&$action, $nextAction, $visitorDetails)
             unset($action['url_prefix']);
         }
 
+        if (array_key_exists('url', $action) && preg_match('/^http:\/\/([^\/]+)\//i', $action['url'], $host)) {
```

Comment:
haven't tested it but might be better using a simple `strpos($url, 'http://')===0` and then use `parse_url` to get the host? Regexes are always tricky to understand and hard to say how well it works etc. Could be also good to add a test if possible. Performance wise it's probably fine as it would be only for HTTP urls.

Reply:
Thought about that solution before, but wanted to avoid two operations for that. But guess most people should have https anyway and so only a strpos would be executed, which should for sure be faster....